docs/en/advanced-concepts/migrating-to-appium-2.0.md

```diff
@@ -84,11 +84,18 @@ To make everyone's lives a bit easier, we've also introduced the option of wrapp
 
 (Of course, each client will have a different way of creating structured capabilities like `appium:options` or other ones that you might have seen such as `goog:chromeOptions`). NB: capabilities that show up in `appium:options` will overwrite capabilities of the same name that show up at the top level of the object.
 
+:bangbang: *Removed Capabilities*
+
+- autoLaunch
```

Comment:
it would be nice to add an explanation how to establish a session without launching an app as typical use case to set `autoLaunch: false`.

e.g.:
> XCUITest driver launches the Home Screen when neither of the `appium:app` or `appium:bundleId` capabilities are provided. On UIAutomator2 driver, neither of the `appium:app`, `appium:appPackage` or `appium:browserName` capabilities are provided then the driver starts from the dashboard. You can launch the application under test via [Activate App](https://appium.io/docs/en/commands/device/app/activate-app/), mobile commands if available or click an icon on the home screen.